plugins/stingray/the_debuginator_plugin/c_api_the_debuginator.cpp

```diff
@@ -292,6 +295,8 @@ void handle_default_input(TheDebuginator* debuginator, unsigned devices) {
 		return;
 	}
 
+	float default_focus_height = 0.3f;
```

Comment:
I must have missed that there was a default (I only spotted the used focus_height which was constant, but I'm misusing that for scroll now so it changes over time).